instance/migrations/0031_openedxinstance_github_pr_url.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.db import models, migrations
+
+
+def migrate_urls(apps, schema_editor):
+    Instance = apps.get_model("instance", "OpenEdXInstance")
+    instances = Instance.objects.exclude(github_pr_number__isnull=True)
+    for instance in instances:
+        instance.github_pr_url = 'https://github.com/{fork}/pull/{number}'.format(
+            fork=getattr(settings, 'WATCH_FORK', 'edx/edx-platform'),
+            number=instance.github_pr_number
+        )
+        instance.save()
+
+
+def reverse_migrate_urls(apps, schema_editor):
+    Instance = apps.get_model("instance", "OpenEdXInstance")
+    instances = Instance.objects.exclude(github_pr_url='')
```

Comment:
Since the integer field on the current model is nullable, would the empty string be correctly converted to a null value? Not sure how the django migrations behave in this case.